docs/understanding-docker.md

```diff
@@ -17,274 +17,287 @@ weight = -90
 # Docker Overview
 
 Docker is an open platform for developing, shipping, and running applications.
-Docker is designed to deliver your applications faster. With Docker you can
-separate your applications from your infrastructure and treat your
-infrastructure like a managed application. Docker helps you ship code faster,
-test faster, deploy faster, and shorten the cycle between writing code and
-running code.
+Docker delivers your applications quickly by separating your applications from
+your infrastructure, so that your infrastructure operates as a managed application.
+Docker's methodologies enable you to ship, test, and deploy code quickly, and to
+shorten the cycle between writing code and running code.
 
-Docker does this by combining kernel containerization features with workflows
-and tooling that help you manage and deploy your applications.
+Docker accomplishes these goals by combining kernel containerization features
```

Comment:
This sentence is awkward "what goals?" You might be able to move this information "combining kernel containerization workflows" to replace "Docker's methodologies" above, which doesn't have a lot of meaning.